docs/bootstrap.md

```diff
@@ -0,0 +1,61 @@
+# Boostrap a new environment
+
+To rebuild the whole Foreman Infrastructure from scratch, these are the steps to get started.
+
+## Build a Puppetserver
+
+Install a minimal EL8 install. Then:
+
+```sh
+dnf install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
+dnf install puppetserver
+. /etc/profile.d/puppet-agent.sh
+puppetserver ca setup --ca-name 'Foreman Puppet CA' --certname $HOSTNAME --subject-alt-names puppet.theforeman.org
+puppet config set --section agent server puppet.theforeman.org
+puppet config set --section main dns_alt_names puppet.theforeman.org
```

Comment:
Shouldn't this one be already there, as the Puppetserver defaults to adding `puppet.$domain`? Or did they stop doing this?

Reply:
Right now the domain is set to `conova.theforeman.org` so I think it wouldn't work well. I also had to add this at runtime because otherwise it would reject requests.

Reply:
Ah right, because the hoster subdomain, yeah ok, I can see that.